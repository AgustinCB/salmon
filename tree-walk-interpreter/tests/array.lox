var array1 = [0; 3];
var array2 = [0, "hola", true];
print array1;
print array2;
print array1[0];
print array2[2];
array1[0] = 2;
print array1[0];