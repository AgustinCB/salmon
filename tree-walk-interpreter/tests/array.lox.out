[ 0, 0, 0, ]
[ 0, hola, true, ]
0
true
2
